includes/Core/McpAdapter.php

```diff
@@ -17,34 +17,13 @@
 /**
  * WordPress MCP Registry - Main class for managing multiple MCP servers.
  */
-class McpAdapter {
+final class McpAdapter {
```

Comment:
1. `final` unless/until we have a specific use case for extending this class means we don't need to worry about the shape of the public methods. See #22 
2. Personally, I'd rename this class `Registry` but since the outdated docs all still reference this class I didn't want to deal with that right now.

Reply:
> This might be the best time to introduce breaking changes.

Great I'll create a new PR once this is merged.

> For the name, I would go with McpAdapterRegistry 

Per https://github.com/WordPress/mcp-adapter/pull/32/files#r2313545529 , I'm now leaning more to `WP/MCP/Core/ServerRegistry` to offset the `WP/MCP/Core/ClientRegistry`. (or Mcp{Client|Server}Registry if you don't think it's redundant). Either way, we'll figure it out on that PR 🚀